extensions/blocks/contact-form/components/jetpack-newsletter-integration-settings.js

```diff
@@ -44,6 +63,191 @@ const NoConsentBlockSettings = () => {
 	);
 };
 
+const pluginStateEnum = Object.freeze( {
+	ACTIVE: 1,
+	INSTALLED: 2,
+	NOT_INSTALLED: 3,
+} );
+
+const CreativeMailPluginErrorState = ( { error } ) => {
+	return (
+		<Notice isDismissible={ false } status="error">
+			{ jetpackCreateInterpolateElement(
+				__(
+					'The plugin failed to install. <b /> Please check the <a>plugin information</a> for detailed requirements.',
+					'jetpack'
+				),
+				{
+					a: (
+						<ExternalLink href="https://wordpress.org/plugins/creative-mail-by-constant-contact" />
+					),
+					b: <span>{ error }</span>,
+				}
+			) }
+		</Notice>
+	);
+};
+
+const CreativeMailPluginIsInstalling = ( { isActivating } ) => {
+	return (
+		<Button
+			isSecondary
+			icon={ <Icon style={ { animation: 'rotation 2s infinite linear' } } icon="update" /> }
```

Comment:
Should we use `<Spinner />` here as well, to remain consistent with the rest of the block editor?

Reply:
The 'update' icon with rotation on to of it. is the same icon/animation that's used on the /wp-admin/plugins.php page when why click on the install button. we tried to replicate that.